Src/AmrCore/AMReX_ErrorList.H

```diff
@@ -442,7 +442,7 @@ std::ostream& operator << (std::ostream& os, const ErrorList& elst);
       {
           AMREX_ASSERT(value.size() > 0);
           m_value.resize(info.m_max_level);
-          for (int i = 0; i < value.size(); ++i) {
+          for (int i = 0; i < m_value.size() && i < value.size(); ++i) {
```

Comment:
An overabundance of caution?  Can't be wrong, but curious what brought that on... -M

Reply:
In Castro I accidentally gave it a vector with max_level+1 values :-)

Reply:
Having the vector of values also gives errors if I'd set max_level=-1, meaning I don't want to use that constraint in that particular run. But max_level=0 works for that purpose as well, since it's strictly less than

Reply:
Yeah that's right, usually with AMR/tagging related things you just set it to max_level = 0 when you want to turn it off.